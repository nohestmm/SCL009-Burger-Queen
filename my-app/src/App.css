.App-logo {
    animation: App-logo-spin infinite 20s linear;
    height: 40vmin;
    pointer-events: none;
}

.navbar {
    overflow: hidden;
    background-color: #fdc123;
    padding: 20px 5px;
    transition: 0.4s;
    position: fixed;
    width: 100%;
    top: 0;
    z-index: 99;
}

.title {
    font-size: 3rem;
    color: #e22450;
    padding-left: 1rem;
    font-family: 'Lobster', cursive;
    margin: 0rem;
    text-align: left;
}

.App-header {
    min-height: 8rem;
    display: flex;
    flex-direction: column;
    align-items: center;
    justify-content: center;
    font-size: calc(10px + 2vmin);
}

body {
    background-color: #059caf;
}

.App-link {
    color: #61dafb;
}

.total {
    background-color: rgba(0, 0, 0, 0.3)
}

.btn-breakfast,
.btn-lunch {
    background-color: #e22450;
    text-align: center;
    margin: 0.5rem;
    padding: 1rem;
    border-radius: 0.5rem;
    border-width: 0px;
    font-family: 'Quicksand', sans-serif;
    font-size: 1.1rem;
    cursor: pointer;
    width: 300px;
    height: 110px;
}

.btn-election {
    background-color: #f07b1c;
    text-align: center;
    margin: 1rem;
    padding: 1rem;
    padding-left: 2rem;
    padding-right: 2rem;
    border-radius: 0.5rem;
    border-width: 0px;
    font-family: 'Quicksand', sans-serif;
    font-size: 1.3rem;
    cursor: pointer;
}

@keyframes App-logo-spin {
    from {
        transform: rotate(0deg);
    }
    to {
        transform: rotate(360deg);
    }
}

.firstcolumn {
    width: 300px;
}

.thirdcolumn {
    margin-left: 50px;
}