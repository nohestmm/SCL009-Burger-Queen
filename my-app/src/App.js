import React from 'react';
import './App.css';
import Nav from './components/Nav';
import ButtonElection from './components/ButtonElect';
import { configure} from 'enzyme';
import Adapter from 'enzyme-adapter-react-16'

class App extends React.Component {
   render() {
      return (
         <>
            <Nav />
            <ButtonElection />
         </>
      );
   }
}


/*
 function App() {
 
return (
    <div className="App">
     
       <Nav/>  
      <ButtonElect/>
     </div>

  );
}
*/



export default App;

