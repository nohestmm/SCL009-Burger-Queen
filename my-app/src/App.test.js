import React from 'react';
import ReactDOM from 'react-dom';
import App from './App';
import { configure, shallow} from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import ButtonElection from './components/ButtonElect';

configure({adapter:new Adapter()})

it('renders without crashing', ()=>{

const clickButton = shallow(<ButtonElection/>);
expect(clickButton).toMatchSnapshot()

            
clickButton.find('button').simulate('click');
expect(clickButton).toMatchSnapshot()


})

it('renders without crashing', () => {
  const div = document.createElement('div');
  ReactDOM.render(<App />, div);
  ReactDOM.unmountComponentAtNode(div);
});
