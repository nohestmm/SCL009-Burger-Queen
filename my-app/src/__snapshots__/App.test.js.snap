// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing 1`] = `
<Fragment>
  <div
    className="App"
  >
    <div
      className="row justify-content-end"
    >
      <div
        className="col-12 col-sm-8 col-md-10 col-lg-8 col-xl-8"
      >
        <button
          className="btn-election"
          onClick={[Function]}
        >
          Desayuno
        </button>
        <button
          className="btn-election"
          onClick={[Function]}
        >
          Almuerzo
        </button>
      </div>
    </div>
  </div>
  <ButtonSelection
    newMenu={Array []}
  />
</Fragment>
`;
